### PR TITLE
Feat/proofpoint/pod/add message parts parsing

### PR DESCRIPTION
**Pull request by Adamowoc**
Add "msgParts" handling to proofpoint parser since we split the messages.

### CODE REVIEW

**Review by squioc — APPROVED** [review]

File: ProofPoint/proofpoint-pod/_meta/smart-descriptions.json [hunk 1]
```diff
@@ -1,63 +1,88 @@
...
+        "relationships": [
+            {
+                "source": "proofpoint.pod.smtp.sender",
+                "target": "proofpoint.pod.smtp.recipients",
+                "type": "sent email to"
+            }
+        ]
+    },
+    {
+        "value": "{email.attachments.file.name}, of type: {email.attachments.file.mime_type} , size: {email.attachments.file.size}bytes, associated with {email.local_id}",
```
Comment (squioc) [inline ProofPoint/proofpoint-pod/_meta/smart-descriptions.json:68]:
```suggestion
        "value": "{email.attachments.file.name}, of type: {email.attachments.file.mime_type} and size: {email.attachments.file.size} bytes, was attached to {email.local_id}",
```

**Review by squioc — APPROVED** [review]

File: ProofPoint/proofpoint-pod/ingest/parser.yml [hunk 2]
```diff
@@ -118,6 +89,39 @@ stages:
...
+                "hash": {
+                  "md5": "{{json_event.message.msgParts.md5}}",
+                  "sha256": "{{json_event.message.msgParts.sha256}}",
+                }
+              }
+            ]
+        proofpoint.pod.urls: >
+            [
+              {% for url in json_event.message.msgParts.urls %}
+              "{{url.url}}",
```
Comment (squioc) [inline ProofPoint/proofpoint-pod/ingest/parser.yml:122]: To be coherent with the removed block.
```suggestion
              {% if url.url | is_url %}"{{url.url}}",{% endif %}
```

**Review by squioc — CHANGES_REQUESTED** [review]

**Review by squioc — DISMISSED** [review]

**Comment by Adamowoc** [comment]
I fixed the bug, removed msgParts from the tests and added a msgParts test.

**Review by squioc — CHANGES_REQUESTED** [review]
Could you merge the stage `set_CTI_fields` in the stage `set_fields_message_parts` and replace, in the jinja expression for `url.original`, `file.name` and `file.hash.*`, `set_fields_for_messages` with `set_fields_for_message_parts`?

**Review by squioc — APPROVED** [review]

**Pull request merged**